src/main/cpp/DragonVision/DragonCamera.h

```diff
@@ -32,8 +32,9 @@ class DragonCamera
         UNKNOWN = -1,
         OFF,
         MACHINE_LEARNING,
-        APRIL_TAG,
-        COLOR_THRESHOLD
+        APRIL_TAG = 0,
+        COLOR_THRESHOLD,
```

Comment:
Can't set numbers in the middle of the enum unless you are creating gaps. unknown = -1
off = 0
april tag = 0
color threshold = 1
tag zoom = 1

That is going to confuse lots of things; in fact, I'm sort of surprised it compiled